cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -744,9 +767,27 @@ public boolean createTimeseries(InsertPlan insertPlan)
     if (unregisteredSeriesList.isEmpty()) {
       return true;
     }
-    logger.debug("Unregisterd series of {} are {}", seriesList, unregisteredSeriesList);
+    logger.debug("Unregistered series of {} are {}", seriesList, unregisteredSeriesList);
 
-    return createTimeseries(unregisteredSeriesList, seriesList, insertPlan);
+    return createTimeseries(unregisteredSeriesList, seriesList, insertPlan, sgNodeVersion);
+  }
+
+  /**
+   * @param partialPath the storage group's partialPath
+   * @return the major version and the minor version of the storage group node
+   */
+  private Pair<Long, Long> getSgNodeMajorVersion(PartialPath partialPath) throws MetadataException {
```

Comment:
change function name as we not only return `MajorVersion`?

Reply:
sure, thanks